include/pacmod_game_control/globals.h

```diff
@@ -40,6 +40,13 @@ enum TurnSignalState
   SIGNAL_HAZARD
 };
 
+enum DoorSignalState
+{
+  SLIDING_DOOR_NEUTRAL,
```

Comment:
What does neutral mean? Is this the transition state when a door is in the process of opening or closing?

Reply:
Yes! Exactly